yarn-project/archiver/src/archiver/archiver_store_test_suite.ts

```diff
@@ -2253,7 +2242,7 @@ export function describeArchiverDataStore(
         block.body.txEffects = await timesParallel(numTxsPerBlock, async (txIndex: number) => {
           const txEffect = await TxEffect.random();
           txEffect.privateLogs = mockPrivateLogs(blockNumber, txIndex);
-          txEffect.publicLogs = mockPublicLogs(blockNumber, txIndex);
+          txEffect.publicLogs = []; // No public logs needed for private log tests
```

Comment:
Shouldn't we test some cases with public logs? Even when we know the implementation handles it well now, we probably want to spec that, especially consider the LogStore processes and reduces logs of both types at the same time

Reply:
There is a new block of tests for `getPublicLogsByTagsFromContract`. Or do you think they are insufficient.